pkg/management/component.go

```diff
@@ -806,10 +806,7 @@ func (c *component) UpdateRealmCustomConfiguration(ctx context.Context, realmNam
 		}
 	}
 	if !match {
-		return http.Error{
-			Status:  400,
-			Message: "Invalid client ID or redirect URI",
-		}
+		return commonhttp.CreateBadRequestError("invalidClientIDOrRedirectURI")
```

Comment:
It would be nice to have invalidParameter.XXX but we would like to have something better than invalidParameter.clientIdOrRedirectURI